pepper-apis/dss-core/src/main/resources/ESprotectedColumns.conf

```diff
@@ -0,0 +1,9 @@
+"columnsWithDefaultValues" : {
```

Comment:
I believe what we need to skip loading into elastic are study specific and we don't need to run through these for All study exports. 
Can we capture these columns by study and export can check only for those studies which has values that need to be skipped. 
If it's hard to get that info in configuration, may be we can add a  database table to capture it.
study_id, alias, stable_id (or whatever is needed)

Reply:
If its in DB, later some point of time we can have study-builder config to populate this table during study configuration too. Also its not something that changes by env and hence can be in DB.